package.json

```diff
@@ -7,7 +7,7 @@
   "license": "BSD-3-Clause",
   "repository": "Polymer/pwa-starter-kit",
   "scripts": {
-    "start": "tsc && polymer serve",
+    "start": "tsc --watch && polymer serve",
```

Comment:
`polymer serve` won't be executed until `tsc --watch` ends, but since `tsc --watch` is watching for file changes, it will never end and so it won't allow `polymer serve` to get executed, so this won't work.

_P.S. You can also use `-w` instead of `--watch`_

Reply:
Ops! Thanks @Dabolus! I forget remove on `&`.

PS: Personally I like to be more explicit when it is written in a file.